docs/help.md

```diff
@@ -220,6 +220,40 @@ correctly with the use of [`chcp`](https://ss64.com/nt/chcp.html) by
 executing in the terminal `chcp 65001`. This is a known limitation of
 Node.js.
 
+It is also possible to change the console encoding from within nodejs using the [ffi-napi](https://www.npmjs.com/package/ffi-napi) module (or any other ffi module you like, [koffi](https://www.npmjs.com/package/koffi) for example).
+
+### Changing encodings from within Nodejs
```

Comment:
An introduction that describes why this would need to be done should be included.